x-pack/legacy/plugins/index_management/public/app/components/mappings_editor/components/document_fields/field_parameters/ignore_malformed.tsx

```diff
@@ -20,7 +20,8 @@ export const IgnoreMalformedParameter = ({ description }: { description?: string
       description
         ? description
         : i18n.translate('xpack.idxMgmt.mappingsEditor.ignoredMalformedFieldDescription', {
-            defaultMessage: 'Whether to ignore malformed data.',
+            defaultMessage:
+              'Malformed data will not be indexed, but other fields in the document will be processed normally. The number of documents that have a malformed field should be contained, or queries on this field will become meaningless.',
```

Comment:
Do you think it's also worth describing the default behavior? I also wasn't sure what "contain" meant. Maybe we can be more direct? Here's my suggestion:

> By default documents that contain the wrong data type for a field will not be indexed. Enable this setting to index these documents normally, but filter out fields that have the wrong data type. Be careful: if you index too many documents this way then queries on this field will become meaningless.

Reply:
By default, documents that contain the wrong data type for a field are not indexed. If enabled, these documents are indexed, but fields with the wrong data type are filtered out. Be careful--if too many documents are indexed this way, queries on the field become meaningless .